addons/edi_notifier/__init__.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 
```

Comment:
Bit of a nit, but an initial blank line in a file seems wrong somehow (this applies to a number of files)?

Reply:
Yes agreed, I used a reg-ex that would remove the line with utf-8, not just empty it. But I didn't consider that some files with this set also left a blank line underneath. Have removed.